scripts/devops/collect_ci_stats.py

```diff
@@ -6,8 +6,13 @@
 from pathlib import Path
 from typing import List
 
+import boto3
+from botocore.auth import SigV4Auth
+from botocore.awsrequest import AWSRequest
 import requests
 
+API_URL = "https://8np7tbux24.execute-api.us-east-1.amazonaws.com/v2/log"
```

Comment:
strange url? why not "api.meshinspector.com"?

Reply:
fixed